From ef3077dbe4052f99b1782d8796846cf4b5765de5 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 6 Jun 2013 13:17:10 +0900
Subject: [PATCH 565/675] LTSI: ARM: shmobile: r8a7778: tidyup generation
 conflict

LTSI 3.4.46 still doesn't have usb_put_phy()
This patch fixup it as old version

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
---
 arch/arm/mach-shmobile/setup-r8a7778.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-shmobile/setup-r8a7778.c b/arch/arm/mach-shmobile/setup-r8a7778.c
index ebd4689..2e39395 100644
--- a/arch/arm/mach-shmobile/setup-r8a7778.c
+++ b/arch/arm/mach-shmobile/setup-r8a7778.c
@@ -31,7 +31,8 @@
 #include <linux/serial_sci.h>
 #include <linux/sh_timer.h>
 #include <linux/pm_runtime.h>
-#include <linux/usb/phy.h>
+#include <linux/usb/otg.h>
+#include <linux/usb.h>
 #include <linux/usb/hcd.h>
 #include <linux/usb/ehci_pdriver.h>
 #include <linux/usb/ohci_pdriver.h>
@@ -381,7 +382,7 @@ void __init r8a7778_add_standard_devices(void)
 
 void __init r8a7778_init_late(void)
 {
-	phy = usb_get_phy(USB_PHY_TYPE_USB2);
+	phy = usb_get_transceiver();
 
 	platform_device_register_full(&ehci_info);
 	platform_device_register_full(&ohci_info);
-- 
1.8.2

